pkg/controller/volume/scheduling/scheduler_binder.go

```diff
@@ -706,7 +742,10 @@ func (b *volumeBinder) checkVolumeProvisions(pod *v1.Pod, claimsToProvision []*v
 	provisionedClaims = []*v1.PersistentVolumeClaim{}
 
 	for _, claim := range claimsToProvision {
-		pvcName := getPVCName(claim)
+		pvcName, err := cache.MetaNamespaceKeyFunc(claim)
+		if err != nil {
+			return false, nil, fmt.Errorf("failed to get pvc name: %v", err)
+		}
```

Comment:
For convenience of debugging, add the claim key in the errorf.

Reply:
added